primitives/state-machine/src/testing.rs

```diff
@@ -101,23 +95,21 @@ impl<H: Hasher, N: ChangesTrieBlockNumber> TestExternalities<H, N>
 		let changes_trie_config = storage.top.get(CHANGES_TRIE_CONFIG)
 			.and_then(|v| Decode::decode(&mut &v[..]).ok());
 		overlay.set_collect_extrinsics(changes_trie_config.is_some());
+		overlay.enable_offchain_indexing();
```

Comment:
I guess it's not possible to move that to the constructor function?

Reply:
IIRC that is totally doable (I did consider it), will try.